### DISCUSSION TITLE
Normalization for scATAC and sctransform

### DISCUSSION THREAD

**Discussion by chrarnold** [stuart-lab/signac#1108] [Q&A] [answered]
Hi,
I am wondering whether sctransform is also applicable to scATAC data, and if not, why not, what are the data modality differences / statistical reasons for it? Has this been tested somewhere, I did not find anything related to it when Googling around. How it performs as compared to TF-IDF?

Thanks for your comments and insights!

**Comment by chrarnold** [comment]
I guess the main problem is sparsity, which scRNA-seq data doesnt have to the extend that scATAC suffers from it, right?

**Comment by timoast** [comment, marked as answer]
SCTransform expects the data to follow a negative binomial or poisson distribution, which is not the case for scATAC data. SCTransform is also trying to normalize the data in a way that accounts for the strong mean-variance relationship in scRNA-seq, which is not something that we observe in scATAC due to the much lower dynamic range.

From a practical perspective also, we want to preserve the sparsity of the data when normalizing scATAC-seq due to the large number of features we have. If we performed a normalization that lost the sparsity, we would have a very large dense matrix that will require a huge amount of memory to hold.
